homework/src/main/java/io/codelex/kontroldarbs/exercise2/Basket.java

```diff
@@ -0,0 +1,57 @@
+package io.codelex.kontroldarbs.exercise2;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Basket<T> {
+
+    private List<T> list;
+    private Integer itemCount;
```

Comment:
Šis teorētiski jau nav nepieciešams, jo var vadīties pēc list izmēra.

Reply:
Šo sapratu jau pēc testa beigām. Novācu itemCount, atstāju list.size() kā parametru pēc kura pārbaudīt.